packages/nimble-components/build/generate-workers/index.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+
+const workersDirectory = path.resolve(__dirname, '../../src/wafer-map/workers');
+
+const matrixRendererBundlePath = path.resolve(
+    __dirname,
+    './dist/bundle/matrix-renderer.js'
+);
+const matrixRendererBundle = fs.readFileSync(matrixRendererBundlePath, 'utf-8');
+const typesDTSPath = path.resolve(__dirname, './dist/esm/types.d.ts');
+const typesDTS = fs.readFileSync(typesDTSPath, 'utf-8');
+const matrixRendererTypesPath = path.resolve(
+    __dirname,
+    './dist/esm/matrix-renderer.d.ts'
+);
+const matrixRendererTypes = fs.readFileSync(matrixRendererTypesPath, 'utf-8');
+
+const workerCode = `export const workerCode = ${JSON.stringify(matrixRendererBundle)};`;
+
+const workerCodeOutPath = path.resolve(workersDirectory, 'worker-code.ts');
+const typesDTSOutPath = path.resolve(workersDirectory, 'types.ts');
+const matrixRendererTypesOutPath = path.resolve(
+    workersDirectory,
+    'matrix-renderer.ts'
+);
+
+prepareDirectory(workersDirectory);
+writeFile(workerCodeOutPath, workerCode);
+writeFile(typesDTSOutPath, typesDTS);
+writeFile(matrixRendererTypesOutPath, matrixRendererTypes);
```

Comment:
Based on the file name, looks like this variable name is for the .ts, not the .d.ts
```suggestion
const workerCodeOutPath = path.resolve(workersDirectory, 'worker-code.ts');
const typesTSOutPath = path.resolve(workersDirectory, 'types.ts');
const matrixRendererTypesOutPath = path.resolve(
    workersDirectory,
    'matrix-renderer.ts'
);

prepareDirectory(workersDirectory);
writeFile(workerCodeOutPath, workerCode);
writeFile(typesDTSOutPath, typesTS);
writeFile(matrixRendererTypesOutPath, matrixRendererTypes);
```